doc.html

```diff
@@ -36,71 +44,96 @@ <h2>2 Terminology</h2>
 
 When used alone, the term "asset_tracking" refers to "asset_tracking ATLAS Service Template". Instead, "asset_tracking service" is equivalent to "an ATLAS Service implementing the asset_tracking ATLAS Service Template".
 
+<h3>2.1 GSIN</h3>
+The Global Shipment Identification Number (GSIN) is the GS1 Identification Key used to identify a logical grouping of logistic units for the purpose of a transport shipment from the seller to the buyer. The unique identifier is comprised of a GS1 Company Prefix, a Shipper Reference, and a Check Digit.
+
+<h3>2.2 SSCC</h3>
+The Serial Shipping Container Code (SSCC) is the GS1 identifier used to identify a logistic unit. This unique identifier is comprised of an Extension Digit, a GS1 Company Prefix, a Serial Reference, and a Check Digit.
+
+<h3>2.3 MAC Address</h3>
+Unique physical identifier of returnable asssets, including beacons and trackers, which are attachd to pallets and containers.
+
+GSIN and SSCC both are unique identifiers in GS1 standard which are generated and assigned to logistic units while a shipment is being created. Whereas, asset's identifier (MAC address) is a unique physical identifier which is hardcoded into the asset and never changes. Therefore, a container or pallet in mupltiple shipments during its lifetime can have multiple GSIN and SSCC, whereas, the returnable asset installed in the pallet or container will have only one MAC address.
+
 <h2>3 Pre-requisites</h2>
 
 A thorough understanding in the following is required for
 both service consumers or service provider implementors:
 <pre>
-    - GeoJSON specifications (<a href="https://geojson.org/">https://geojson.org/</a>)
+    - GeoJSON specifications (<a href="https://geojson.org/">https://geojson.org/</a>) 
     - ISO8601 representation of dates and times
       (<a href="https://www.iso.org/iso-8601-date-and-time-format.html">https://www.iso.org/iso-8601-date-and-time-format.html</a>)
 </pre>
 <h2>4 asset_tracking Usage Scenarios</h2>
 
-TO BE DONE
+The following are some examples of how the asset_tracking service can be used:
+<h3>4.1 Product Monitoring</h3>
+The "asset_tracking" service template can provide real-time temperature and humidity data for a product shipped using a pallet. In this case, the service can be used to see if products are in a good environment in terms of temperature and humidity.
+
+<h3>4.2 Product Tracking</h3>
+Clients can get general information regarding the shipping status of the their products. For example, the client can get the current location of the product, the time it was shipped, and the time it was received.
```

Comment:
I have not seen anything in the specifications that would indicate the time a product was shipped or received (no notion of status)

Reply:
Will be added as another endpoint in the future. This description will be deleted for now.